src/js/types/deck.ts

```diff
@@ -20,6 +20,7 @@ export interface Card {
     type_line?: string;
     oracle_id?: string;
   };
+  color_identity: color[] | ["C"];
```

Comment:
FYI, this is not correct.

This type is for the deck entry object on the Scryfall deckbuilder, but `color_identity` is a property that exists on the Scryfall API response, not the deck entry object.

Reply:
Okay, I removed that line and the color type now